src/mongodb/testHelpers/helper.ts

```diff
@@ -0,0 +1,25 @@
+import { MongoMemoryServer } from "mongodb-memory-server";
+import mongoose from "mongoose";
+
+export const createCleanDatabase = async () => {
+  const mongod = await MongoMemoryServer.create();
+  const uri = mongod.getUri();
+  (global as any).__MONGO_INSTANCE = mongod;
+  await cleanDatabase(uri);
+  process.env.MONGODB_URI = uri;
+};
+
+export const cleanDatabase = async (uri: string) => {
+  if (uri.includes("mongodb://127.0.0.1")) {
+    await mongoose.connect(uri);
+    await mongoose.connection.dropDatabase();
+    await mongoose.disconnect();
+  } else {
+    throw new Error("Not a test database");
+  }
+};
+
+export const stopDatabase = async () => {
+  const mongod = (global as any).__MONGO_INSTANCE;
+  await mongod.stop();
```

Comment:
Does this want a localhost check as well? Not sure what would happen if you ran this against a non-test database, but it feels like an easy thing to guard against.

Reply:
I think this is fine because it's just the connection to the database itself. I guarded the other one because it was making changes to the database.

Reply:
On rereading, yeah, it can only be the in-memory instance, can't it... Never mind 😁